cli/attestation_policy.go

```diff
@@ -226,6 +228,52 @@ func (cli *CLI) NewDownloadGCPOvmfFile() *cobra.Command {
 	}
 }
 
+func (cli *CLI) NewAzureAttestationPolicy() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "azure",
+		Short:   "Get attestation policy for Azure CVM",
+		Example: `azure <azure_maa_token_file> <product_name>`,
```

Comment:
we'll need this token file on the cloud as well so we need to send it simillar to attestation when agent boots up https://github.com/ultravioletrs/cocos/blob/dffefc45d83e7ebfde24b12842d8b0299b8ccd69/cmd/agent/main.go#L178-L192